src/mindMapPreviewWebview.ts

```diff
@@ -17,6 +23,7 @@ class MindMapPreview {
                 ]
             }
         );
+        console.log("path", path.join(context.extensionPath, 'html'));

```

Comment:
Debug logs like this should not appear in the master branch. Please delete it. Thanks.